site/src/component/SearchFilters/SearchFilters.tsx

```diff
@@ -0,0 +1,157 @@
+import { FC } from 'react';
+import './SearchFilters.scss';
+
+import CheckIcon from '@mui/icons-material/Check';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import ListItemText from '@mui/material/ListItemText';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import { Icon } from '@mui/material';
+
+import { useAppSelector } from '../../store/hooks';
+
+import { FilterOptions, levels, geCategories } from '../../helpers/searchFilters.ts';
+
+interface SearchFiltersProps {
+  selectedFilters: FilterOptions;
+  updateSelectedFilters: (value: FilterOptions) => void;
+}
```

Comment:
tbh i think filters belongs in the `searchSlice` rather than as used state. This would make it easier to read from/write to, esp since we have nested components.